src/types.ts

```diff
@@ -96,7 +96,7 @@ export interface EventCategory {
 }
 
 export interface ScrapeOptions {
-  proxy?: AxiosProxyConfig;
+  axiosOptions?: AxiosRequestConfig;
```

Comment:
I decided to introduce a breaking change here which is not really neccessary but it gives us bigger flexibility in the future if we decided to introduce library specific options

Reply:
Sorry been busy, will release